src/pages/Home.tsx

```diff
@@ -9,41 +9,35 @@ export const Home = () => {
     window.location.assign(link);
   };
 
-  const screenIsSmall= () => {
-    if (window.innerWidth <= 1536) {
-      return true;
-    } else {
-      return false
-    }
-  }
-
   return (
     <>
       <main id="mainHome" className="container">
-        <section className="container d-flex flex-column justify-content-between">
-          <div className="row d-flex justify-content-between mainHomeRow1">
-            <div className="col-sm-6">
+        <section className="container d-flex flex-column justify-content-between mainHomeContainer">
```

Comment:
convert all css custom class from camelCase to kebab-case. On this way we follow the same convention than bootstrap. add the prefix "nina" to identify nina custom classes. For example:

`nina-main-home-container`.

don't use numbers to differentiate classes with the same name.  For example:

`mainHomeRow1`

Delete classes that are duplicated.

Reply:
done